spec/controllers/api/v1/patients_controller_spec.rb

```diff
@@ -72,7 +72,7 @@
 
         patients_payload.each do |updated_patient|
           db_patient = Patient.find(updated_patient['id'])
-          expect(db_patient.attributes.with_int_timestamps.except('address_id', 'updated_on_server_at'))
+          expect(with_payload_keys(db_patient.attributes).with_int_timestamps.except('address_id'))
```

Comment:
Where is this expecting to find `with_payload_keys` defined? Isn't it an instance method for the patient model?